docs/general-guides/governmentpay.md

```diff
@@ -0,0 +1,105 @@
+---
+sidebar_position: 2
+---
+
+# Government Pay
+
+## Introduction
+Welcome to our guide on the Government Pay Structure our concise roadmap through the complexities of how public sector employees are compensated. Whether you're an aspiring civil servant, a current government worker, or a curious citizen, this guide provides clear insights into the principles, classifications, and nuances of government pay. 
+Join us as we unravel the essentials, demystifying the mechanisms that ensure fairness and transparency in public sector remuneration.
+
+## Level 0: Commission
+
+This level within the pay structure consists of all entry level government employees. These indiiviudals get paid based off of work completed determined by a higher level of management.
+
+### Cabinet
+- President
+- Vice President
+- Secretary
+- Deputy Secretary
+- Chief of Staff
+- Special Advisors
+- Press Advisor
+
+### Department of State
+- Electoral Officer
+- Auditor
+- Ambassador
+- Foreign Officer
+
+### Department of Interior
+- Ranger
+- Environment Manager
+- Supply Manager
+
+### Department of Legal Affairs
+- State Prosecutor
+
+### Department of Public Affairs
+- Media Advisor
+- Event Coordinator
+- Media Manager
+- Event Manager
+- Tour Manager
+
+### Department of Construction and Transportation
+- Construction Manager
+- Inspection Manager
+- Constructor
+- Building Inspector
+
+### Department of Education
+- Education Manager
+- Archivist
+- Guide
+- Archive Manager
+
+### Department of Commerce
+- Economist
+
+## Level 1: $45/15 minutes
+
+This level within the pay structure consists of *Leaders of Government Branches; Significant Responsibility, management, and governance*.
```

Comment:
No point italicising everything. I would reword this to:
"This level within the pay structure consists of leaders and upper management of government branches"